README.md

```diff
@@ -61,9 +61,9 @@ module.watch(require("./module"), {
   // The keys of this object literal are the names of exported symbols.
   // The values are setter functions that take new values and update the
   // local variables.
-  default: value => { a = value; },
-  b: value => { b = value; },
-  c: value => { d = value; },
+  default(value) { a = value; },
+  b(value) { b = value; },
```

Comment:
Here's an example of why I avoided method shorthand before, even though it seems appealing at first. The assignment to `b` refers to the method name `b` here, not the outer variable `b`. Since this is the common case (`import { b } from ...`), I don't think we should use a syntax that works only for the uncommon case (`import { b as c } from ...`). Can we revert this commit?